data/World/Water Temple MQ.json

```diff
@@ -18,6 +34,74 @@
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 3": "
+                can_use(Zora_Tunic) and
+                ((logic_water_mq_central_pillar and can_use(Fire_Arrows)) or
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 4": "
+                can_use(Zora_Tunic) and
+                ((logic_water_mq_central_pillar and can_use(Fire_Arrows)) or
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 5": "
+                can_use(Zora_Tunic) and
+                ((logic_water_mq_central_pillar and can_use(Fire_Arrows)) or
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 6": "
+                can_use(Zora_Tunic) and
+                ((logic_water_mq_central_pillar and can_use(Fire_Arrows)) or
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 7": "
+                can_use(Zora_Tunic) and
+                ((logic_water_mq_central_pillar and can_use(Fire_Arrows)) or
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 8": "
+                can_use(Zora_Tunic) and
+                ((logic_water_mq_central_pillar and can_use(Fire_Arrows)) or
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 9": "
+                can_use(Zora_Tunic) and
+                ((logic_water_mq_central_pillar and can_use(Fire_Arrows)) or
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 10": "
+                can_use(Zora_Tunic) and
+                ((logic_water_mq_central_pillar and can_use(Fire_Arrows)) or
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 11": "
+                can_use(Zora_Tunic) and
+                ((logic_water_mq_central_pillar and can_use(Fire_Arrows)) or
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 12": "
+                can_use(Zora_Tunic) and
+                ((logic_water_mq_central_pillar and can_use(Fire_Arrows)) or
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 13": "
+                can_use(Zora_Tunic) and
+                ((logic_water_mq_central_pillar and can_use(Fire_Arrows)) or
+                    (can_use(Dins_Fire) and can_play(Song_of_Time)))",
+            "Water Temple MQ Central Pillar Lower Crate 14": "
```

Comment:
Each of these central pillar crates (except maybe the last two) in a vacuum would be safe under FTR. The problem is that there's kind of a lot of them down there. You'd maybe bonk 2-3 per trip and have to die and come back to get the rest. Kind of an awkward case. Probably the requiring the tunic always is best here, if only because FTR is such a commonly used trick. (Maybe this room informs out decision about Fire Temple a bit, I dunno.)